src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -363,7 +368,7 @@ private int loadInternal (Table table) throws Exception {
         // When outputting text, accumulate transformed strings to allow skipping rows when errors are encountered.
         // One extra position in the array for the CSV line number.
         String[] transformedStrings = new String[cleanFields.length + 1];
-
+        boolean tableHasConditions = table.hasConditionalRequirements();
```

Comment:
Can we rename this to `tableHasConditionalRequirements`?

Reply:
Renamed. Thanks.